pyproject.toml

```diff
@@ -60,14 +42,29 @@ select = ["E", "F", "W"]
 ignore = [
     # whitespace before :
     "E203",
+    # line-too-long (due to Copyright header)
+    "E501",
 ]
 fixable = ["ALL"]
 exclude = [
     # TODO: Remove this in a follow-up where we fix __all__.
     "__init__.py",
 ]
-line-length = 88
-
+line-length = 79
+[tool.ruff.format]
+exclude = [
+    "thirdparty",
+    ".eggs",
+    ".git",
+    ".hg",
+    ".mypy_cache",
+    ".tox",
+    ".venv",
+    "_build",
+    "buck-out",
+    "build",
+    "dist",
+]
```

Comment:
We really don't need all these excludes. All of these are in the [default exclude set](https://docs.astral.sh/ruff/settings/#exclude) except for "thirdparty" and "build", and neither of those exist in the source tree. Let's try removing them and modify if needed. But we _should_ leave a blank line between sections, which my GitHub suggestion refuses to show...
```suggestion

```